linux_os/guide/services/mail/postfix_client/postfix_network_listening_disabled/tests/wrong.fail.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 # packages = postfix
 
-echo "inet_interfaces = some_different_interface,loopback-only" > /etc/postfix/main.cf
```

Comment:
I see value in testing if a correct substring is not mistaken as a correct configuration, the rule description mentions that the only value configured should be the one from the variable.

Reply:
I see that the original test "tested" two things at the same time, so it could be put into a test scenario of its own.

Reply:
That's a good point. I have added a new test scenario similar to the previous one.

Unfortunately, in the new test scenario, I have to start postfix first and then add the configuration. It's because having `inet_interfaces = some_different_interface,loopback-only` requires to have interface called `loopback-only`. Without that, you can't start postfix.